components/usage-api/usage/v1/billing.proto

```diff
@@ -80,14 +78,10 @@ message CreateStripeCustomerResponse {
 
 message CreateStripeSubscriptionRequest {
   string attribution_id = 1;
-  string stripe_intent_id = 2;
-
-  int32 usage_limit = 3;
+  string customer_id = 2;
 }
 
 message CreateStripeSubscriptionResponse {
```

Comment:
Why does the subscription create not return anything?

Reply:
As discussed, the necessary params are put back.